src/cap32.cpp

```diff
@@ -3540,6 +3563,14 @@ int cap32_main (int argc, char **argv)
                            audio_resume();
                            break;
 
+                        case CAP32_SCRNDUMP:
+#ifdef HAVE_PNG
+                           CPC.paused |= 1;
```

Comment:
Pausing is not necessary. Caprice32 is not multithreaded so nothing else will happen during dumpScreen.